test/ffmpeg-vaapi/transcode/avc.py

```diff
@@ -0,0 +1,171 @@
+###
+### Copyright (C) 2018-2019 Intel Corporation
+###
+### SPDX-License-Identifier: BSD-3-Clause
+###
+
+from ....lib import *
+from ..util import *
+from .transcoder import TranscoderTest
+
+spec = load_test_spec("avc", "transcode")
+class to_avc(TranscoderTest):
+  @slash.parametrize(*gen_transcode_parameters(spec, "avc", "hwhw"))
+  @platform_tags(set(AVC_DECODE_PLATFORMS)&set(AVC_ENCODE_PLATFORMS))
+  def test_hwhw(self, case):
+    vars(self).update(spec[case].copy())
+    vars(self).update(
+      srcformat    = 'avc',
+      dstformat    = 'avc',
```

Comment:
`src_codec`, `dst_codec`, instead?

Reply:
used these for reference yuv naming convention and removed now as reference yuv derived from source file